src/govuk/components/skip-link/skip-link.test.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-env jest */
+
+const configPaths = require('../../../../config/paths.json')
+const PORT = configPaths.ports.test
+
+const baseUrl = 'http://localhost:' + PORT
+
+describe('/examples/template-default', () => {
+  describe('skip link', () => {
+    beforeAll(async () => {
+      await page.goto(`${baseUrl}/examples/template-default`, { waitUntil: 'load' })
+      await page.keyboard.press('Tab')
+      await page.click('.govuk-skip-link')
```

Comment:
Would it make more sense to use `page.keyboard.press('Enter')`, as that's a better simulation of what someone navigating using the keyboard would do?

Reply:
@36degrees Could you elaborate on this, I don't think just pressing 'Enter' will navigate to the skip link?

Reply:
It should do, once the link is focussed by pressing tab? I'd expect in most cases someone who is relying on the skip link and navigating the site using their keyboard isn't going to tab to the skip link only then to use their mouse to click it.

Reply:
Oh I see what you mean. Yes, that seems like a better simulation. I've switched it up.